source/includes/wc-analytics/_introduction.md

```diff
@@ -0,0 +1,272 @@
+
+**Please note that to access the `wc-analytics` namespace, the version must be specified as `wc-analytics` in the libraries.**
+
+```javascript
+// Install:
+// npm install --save @woocommerce/woocommerce-rest-api
+
+// Setup:
+const WooCommerceRestApi = require("@woocommerce/woocommerce-rest-api").default;
+// import WooCommerceRestApi from "@woocommerce/woocommerce-rest-api"; // Supports ESM
+
+const WooCommerce = new WooCommerceRestApi({
+  url: 'http://example.com', // Your store URL
+  consumerKey: 'consumer_key', // Your consumer key
+  consumerSecret: 'consumer_secret', // Your consumer secret
+  version: 'wc-analytics' // WooCommerce WP REST API version
+});
+```
+
+```php
+<?php
+// Install:
+// composer require automattic/woocommerce
+
+// Setup:
+require __DIR__ . '/vendor/autoload.php';
+
+use Automattic\WooCommerce\Client;
+
+$woocommerce = new Client(
+    'http://example.com', // Your store URL
+    'consumer_key', // Your consumer key
+    'consumer_secret', // Your consumer secret
+    [
+        'wp_api' => true, // Enable the WP REST API integration
+        'version' => 'wc-analytics' // WooCommerce WP REST API version
+    ]
+);
+?>
+```
+
+```python
+# Install:
+# pip install woocommerce
+
+# Setup:
+from woocommerce import API
+
+wcapi = API(
+    url="http://example.com", # Your store URL
+    consumer_key="consumer_key", # Your consumer key
+    consumer_secret="consumer_secret", # Your consumer secret
+    wp_api=True, # Enable the WP REST API integration
+    version="wc-analytics" # WooCommerce WP REST API version
+)
+```
+
+```ruby
+# Install:
+# gem install woocommerce_api
+
+# Setup:
+require "woocommerce_api"
+
+woocommerce = WooCommerce::API.new(
+  "https://example.com", # Your store URL
+  "consumer_key", # Your consumer key
+  "consumer_secret", # Your consumer secret
+  {
+    wp_api: true, # Enable the WP REST API integration
+    version: "wc-analytics" # WooCommerce WP REST API version
+  }
+)
+```
+
+<aside class="notice">
+	Use the tabs in the top-right corner of this page to see how to install and use each library.
+</aside>
+
+### Third party libraries ###
+
+- [Java](https://github.com/icoderman/wc-api-java) Library
+- [.NET](https://github.com/XiaoFaye/WooCommerce.NET) Library
+- [Android](https://github.com/gilokimu/WooDroid) Library
+
+<aside class="notice">
+	Note that we don't offer support for third party libraries, so if you have questions about how use any of this libraries you should contact the respective authors.
+</aside>
+
+### Tools ###
+
+Some useful tools you can use to access the API include:
+
+- [Insomnia](https://insomnia.rest) - Cross-platform GraphQL and REST client, available for Mac, Windows, and Linux.
+- [Postman](https://www.getpostman.com/) - Cross-platform REST client, available for Mac, Windows, and Linux.
+- [RequestBin](https://requestbin.com) - Allows you test webhooks.
+- [Hookbin](https://hookbin.com/) - Another tool to test webhooks.
```

Comment:
These two URLs will be redirected to another same URL. Maybe the original links aren't available anymore.